### PR TITLE
feat: refresh user on page load

### PR DESCRIPTION
**Pull request by ThibaultNocchi**

### CODE REVIEW

**Review by ferferga — CHANGES_REQUESTED** [review]
Why not do this in the plugin initialization? In the auth plugin constructor, for instance.

**Comment by ThibaultNocchi** [comment]
@ferferga It's implemented now in the constructor :)

File: frontend/src/plugins/remote/auth/index.ts [hunk 1]
```diff
@@ -197,6 +198,22 @@ class RemotePluginAuth {
...
 
+  /**
+   * Refreshes the current user infos, to fetch a new picture for instance
+   */
+  public async refreshCurrentUserInfo(): Promise<void> {
+    if (!isNil(this.currentUser) && !isNil(this.currentServer)) {
+      const api = useOneTimeAPI(
+        this.currentServer.PublicAddress,
+        this.getUserAccessToken(this.currentUser)
+      );
```
Comment (ferferga) [inline frontend/src/plugins/remote/auth/index.ts:209]: If there's an user logged in, you don't need the `OneTimeApi`, you can use the global api instance. In fact if there's no global api defined, we can assume there's no current user/server. Check [this](https://github.com/jellyfin/jellyfin-vue/blob/master/frontend/src/plugins/remote/sdk/index.ts#L17-L51)

**Review by ferferga — CHANGES_REQUESTED** [review]
This made me realize that we could probably simplify our checks in a lot of places, but that's for another time lol

File: frontend/src/plugins/remote/auth/index.ts (hunk 1 above)
Comment (ferferga) [reply frontend/src/plugins/remote/auth/index.ts:209]: Fair enough, this reminds me that I need to add a way to debug circular dependencies in Vite.

**Review by ferferga — APPROVED** [review]

**Pull request merged**